keepsorted/keep_sorted_test.go

```diff
@@ -168,6 +168,95 @@ baz
 foo
 // keep-sorted-test end`,
 		},
+		{
+			name: "sticky comment with prefix to id",
```

Comment:
Could you put this over into a golden test? https://github.com/google/keep-sorted/blob/main/goldens/comments.in

that's also probably a pretty good spot to assert that unconventional comment leaders don't work

Reply:
Done.